cmd/eksctl-anywhere/cmd/deletecluster.go

```diff
@@ -82,8 +82,8 @@ func (dc *deleteClusterOptions) validate(ctx context.Context, args []string) err
 	}
 
 	kubeconfigPath := getKubeconfigPath(clusterConfig.Name, dc.wConfig)
-	if !validations.FileExistsAndIsNotEmpty(kubeconfigPath) {
-		return kubeconfig.NewMissingFileError(kubeconfigPath)
+	if err := validations.KubeConfigFile(kubeconfigPath); err != nil {
```

Comment:
nit: I feel like I would just call `kubeconfig.ValidateFile` here directly and avoid that var in `validations`
Tbh, `validations` as a package sounds too broad (unless it's for validation tooling or something like that). I like more your idea of having the validation on an specific package for that domain

Reply:
I'd be glad to do that. I did it the way I did to try to maintain the pattern, cuz I didn't know a whole lot about the validation package.